app/Http/Kernel.php

```diff
@@ -32,6 +32,7 @@ class Kernel extends HttpKernel
 	 */
 	protected $middlewareGroups = [
 		'web' => [
+			'installation:complete',
```

Comment:
why was it NOT necessary before ?

Reply:
I don't know :grinning: I haven't wrote the web installer. I only have consolidated the middlewares once.

It make sense, that the routes inside the web group should only be accessible after a complete installation, but I always frightened of changes like these, because I cannot assess their impact. Maybe there is a route inside the web group which is needed during installation? Of course it should not.

So the question was rather did you run into a bug and then you changed this or did you change it, because it caught your eye and you changed it, because it should be changed.

Reply:
I didn't see any bug or behavior change. Can this be resolved?